src/sdk/src/Cli/Microsoft.DotNet.Cli.Utils/MSBuildForwardingAppWithoutLogging.cs

```diff
@@ -135,6 +138,26 @@ public void EnvironmentVariable(string name, string? value)
         }
     }
 
+    public static string GetHostServerPath(bool createDirectory)
+    {
+        // If the path is set from outside, reuse it.
+        var hostServerPath = Env.GetEnvironmentVariable(BuildServerUtility.DotNetHostServerPath);
+        if (string.IsNullOrWhiteSpace(hostServerPath))
+        {
+            // Otherwise, construct a directory path under temp.
+            string baseDirectory = PathUtility.GetUserRestrictedTempDirectory();
+            hostServerPath = Path.Join(baseDirectory, "dotnet", "server", Product.TargetFrameworkVersion);
```

Comment:
I'm using `Product.TargetFrameworkVersion` from the sdk but that didn't flow to vmr yet, hence the build errors.